_includes/macros/activity-snippets.html

```diff
@@ -42,7 +42,7 @@
 
 {# ==========================================================================
 
-   _activity_snippet()
```

Comment:
I don't think it's necessary. In javascript we rely upon the underscore to denote something as private. It should be the same in Jinja. `Macros and variables starting with one or more underscores are private and cannot be imported.`